lib/ui/screens/menu_screen.dart

```diff
@@ -1,146 +1,148 @@
+                  ),
+                  const SizedBox(height: 8.0),
+                  MenuItem(
+                    leadingIcon: Icons.phone_outlined,
+                    text: AppLocalizations.of(context)!.contactCarrier,
+                    onPressed: () async =>
+                        await HelperService.makePhoneCall('123456789'),
+                  )
+                ],
+              ),
+              _buildMenuSection(
+                AppLocalizations.of(context)!.saveLunch,
+                [
+                  MenuItem(
+                    leadingIcon: Icons.textsms_outlined,
+                    text: AppLocalizations.of(context)!.feedback,
+                  ),
+                  const SizedBox(height: 8.0),
+                  MenuItem(
+                    leadingIcon: Icons.language,
+                    text: AppLocalizations.of(context)!.about,
+                    onPressed: () async {
+                      await _openUrlInBrowser(ZOStrings.zjUrl);
+                    },
+                  ),
+                  const SizedBox(height: 8.0),
+                  MenuItem(
+                    leadingIcon: Icons.volunteer_activism_outlined,
+                    text: AppLocalizations.of(context)!.sponsors,
+                    onPressed: () async {
+                      await _openUrlInBrowser(ZOStrings.zjUrl);
+                    },
+                  ),
+                ],
+              ),
+              _buildMenuSection(
+                AppLocalizations.of(context)!.more,
+                [
+                  MenuItem(
+                    leadingIcon: Icons.star_border,
+                    text: AppLocalizations.of(context)!.rate,
+                  ),
+                  const SizedBox(height: 8.0),
+                  MenuItem(
+                    leadingIcon: Icons.security,
+                    text: AppLocalizations.of(context)!.privacyProtection,
+                    onPressed: () async =>
+                        await _openUrlInBrowser(ZOStrings.zjUrl),
+                  ),
+                  const SizedBox(height: 8.0),
+                  MenuItem(
+                    leadingIcon: Icons.text_snippet_outlined,
+                    text: AppLocalizations.of(context)!.termsOfUse,
+                    onPressed: () async =>
+                        await _openUrlInBrowser(ZOStrings.zjUrl),
+                  ),
+                ],
+              ),
+              MenuButton(
+                text: AppLocalizations.of(context)!.signOut,
+                icon: Icons.logout,
+                onPressed: () async {
+                  await authService.signOut();
+                  if (context.mounted) {
+                    context.router.replace(const LoginRoute());
+                  }
+                },
+              ),
+              const SizedBox(height: GapSize.xl),
+            ],
+          ),
+        ),
+      ),
+    );
+  }
+
+  Widget _buildMenuSection(String label, List<Widget> menuItems) {
```

Comment:
Bylo by fajn to přetáhnout jako stateless widget, klidně private

Reply:
Done